<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Biblioteca</class>
 <widget class="QMainWindow" name="Biblioteca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Biblioteca</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEsci">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>550</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Esci</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpResoconto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Resoconto dei materiali</string>
    </property>
    <widget class="QLabel" name="lblUltimoArrivo">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ultimo arrivo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDataArrivo">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpBuste">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Buste</string>
     </property>
     <widget class="QLabel" name="lblRosse">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rosse</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBlu">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Blu</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVerdi">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Verdi</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBianche">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bianche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRosse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 0, 0, 50);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBlu">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(9, 47, 252, 50);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtVerdi">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(3, 251, 63, 50);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBianche">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="grpMastelli">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Mastelli</string>
     </property>
     <widget class="QLabel" name="lblMastelliUmido">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Umido</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMastelliVetro">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vetro</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtMastelliUmido">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtMastelliVetro">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="grpCalendari">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>120</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Calendari</string>
     </property>
     <widget class="QLineEdit" name="txtCalendari">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpArrivo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestione Arrivo</string>
    </property>
    <widget class="QPushButton" name="btnAggiungi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Aggiungi</string>
     </property>
    </widget>
    <widget class="QTableView" name="tblArrivo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>451</width>
       <height>121</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btnElimina">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Elimina</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkBtnConferma">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>260</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Conferma Arrivo</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCategoria">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Categoria</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTipologia">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipologia</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDimensione">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dimensione</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCategoria">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTipologia">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDimensione">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblQuantita">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantità</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxQuantita">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Arrivo</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEditArrivo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPulisci">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Pulisci</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVaiArrivi">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Vai a gestione arrivi</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
